frontend-admin/src/pages/vans/vans-page.tsx

```diff
@@ -28,7 +30,7 @@ const VanPage: React.FC = () => {
   const createVan = async (vanData: VanData) => {
     // Convert the routeId to an integer
     try {
-      await fetch('http://localhost:8000/vans/', {
+      await fetch(baseUrl + '/vans/', {
```

Comment:
Nit: Inconsistent use of format strings and string concat

Reply:
sure, we can fix